test/e2e/route_advertisements.go

```diff
@@ -972,7 +972,7 @@ var _ = ginkgo.DescribeTableSubtree("BGP: isolation between advertised networks"
 					// pod -> node traffic should use the node's IP as the source for advertised UDNs.
 					return clientPod.Name, clientPod.Namespace, net.JoinHostPort(nodeIP, fmt.Sprint(hostNetworkPort)) + "/clientip", clientNodeIP, false
 				}),
-			ginkgo.Entry("UDN pod to the same node nodeport service in default network should work (should it? :)...)",
+			ginkgo.Entry("UDN pod to the same node nodeport service in default network should not work",
```

Comment:
spoke to @trozet 
action item is open an issue and do a FIXME for this.

Reply:
https://github.com/ovn-kubernetes/ovn-kubernetes/issues/5410
added a FIXME comment to it